farmdata2_modules/fd2_tabs/fd2_example/cypress/integration/componentTesting.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Field and Crop Dropdowns', () => {
+  beforeEach(() => {
+          //cy.login('manager1', 'farmdata2')
+          cy.visit('/farm/fd2-example/componentTesting')
+  })
+  context('Crop Dropdown', () => {
+    it('accepts input from search bar', () => {
+      cy.get('crop-picker-element')
+        .type('lettuce')
+        .should('have.value', 'lettuce')
+    })
+  })
+})
```

Comment:
Looks like this is left from the preliminary testing.  If so it should be reomved.